weak field

fuhok85 opened the pull request:
Some of the references are reviews. 
In that case I added the bibtex to multiple subtopics, if it covers them

fuhok85 committed:
Update tables.json

fuhok85 committed:
Update tables.json

duetosymmetry commented:
Thanks, @fuhok85, for sorting the refs! I squashed your two commits into one, rebased from master, and edited the commit message to be more descriptive. See eca8aa2 for the final commit.

fuhok85 commented:
Thanks Leo, 

and sorry for the name-misundesranding.. 
I should have told you about the Chinese nickname :)

Write me for any other duty!

Cheers
A  

> On 25 Nov 2017, at 19:31, Leo C. Stein <notifications@github.com> wrote:
> 
> Thanks, @fuhok85 <https://github.com/fuhok85>, for sorting the refs! I squashed your two commits into one, rebased from master, and edited the commit message to be more descriptive. See eca8aa2 <https://github.com/duetosymmetry/testing-gr-tables/commit/eca8aa2e769e45c94f010fee2205b769017cc142> for the final commit.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/duetosymmetry/testing-gr-tables/pull/17#issuecomment-346960954>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AgR0_pD4gNzAxb_a1chwjZYB2MfU9anpks5s6Gr2gaJpZM4Qmonk>.
>
